chsdi/templates/htmlpopup/atlas_basisgebiete.mako

```diff
@@ -1,6 +1,7 @@
 <%inherit file="base.mako"/>
 
 <%def name="table_body(c, lang)">
+<% c['stable_id'] = True %>
```

Comment:
Introduced because data has a stable ID. The or '-' is not needed since it's the ID ;)
So this one is OK